style.css

```diff
@@ -550,6 +546,10 @@ textarea:focus {
     background-position-x: 510px;
   }
 
+  .common-li {
+    border-bottom: none;
+  }
```

Comment:
- [ ] Great job 👍 . As the previous reviewer mentioned kindly add the `background-image` as seen in the Figma design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/191970046-1d74af80-8b69-47e1-b1c4-7e5fe9e2f4a7.png)  |  ![image](https://user-images.githubusercontent.com/74814780/191969969-feaf6915-4334-4aa1-b87c-832efbcbd65c.png)